torch/jit/__init__.py

```diff
@@ -133,15 +197,28 @@ def load(f, map_location=None, _extra_files=DEFAULT_EXTRA_FILES_MAP):
             torch.jit.load(buffer)
 
             # Load all tensors onto CPU, using a device
+            buffer.seek(0)
             torch.jit.load(buffer, map_location=torch.device('cpu'))
 
             # Load all tensors onto CPU, using a string
+            buffer.seek(0)
             torch.jit.load(buffer, map_location='cpu')
 
             # Load with extra files.
-            files = {'metadata.json' : ''}
-            torch.jit.load('scriptmodule.pt', _extra_files = files)
-            print(files['metadata.json'])
+            extra_files = torch._C.ExtraFilesMap()
```

Comment:
use `DEFAULT_EXTRA_FILES_MAP` here instead ?

Reply:
I don't think it should, it's used as a default arg for `load` so if it gets mutated it will affect all future calls to load